tests/test_polyMap.py

```diff
@@ -41,6 +46,77 @@ def test_PolyMapUnidirectional(self):
             self.assertAlmostEqual(xn, xo)
             self.assertAlmostEqual(yn, yo)
 
+        pinRoundTrip = pm.tranInverse(pout)
+        npt.assert_allclose(pin, pinRoundTrip, atol=1e-4)
+
+    def test_polyMapAtributes(self):
+        coeff_f = np.array([
+            [1.2, 1, 2, 0],
+            [-0.5, 1, 1, 1],
+            [1.0, 2, 0, 1],
+        ])
+        pm = astshim.PolyMap(coeff_f, 2, "IterInverse=1, NIterInverse=6, TolInverse=1.2e-7")
+        self.assertIsInstance(pm, astshim.Object)
+        self.assertIsInstance(pm, astshim.Mapping)
+        self.assertIsInstance(pm, astshim.PolyMap)
+        self.assertEqual(pm.getNin(), 2)
+        self.assertEqual(pm.getNout(), 2)
+        self.assertEqual(pm.getNiterInverse(), 6)
+        self.assertAlmostEqual(pm.getTolInverse(), 1.2E-7)
+        self.assertTrue(pm.hasTranForward())
+        self.assertTrue(pm.hasTranInverse())
+
+        pin = np.array([
+            [1.0, 0.0],
+            [2.0, 1.0],
+            [3.0, 2.0],
+        ])
+        pout = pm.tranForward(pin)
+        for (xi, yi, xo, yo) in zip(pin[:, 0], pin[:, 1], pout[:, 0], pout[:, 1]):
+            xn = 1.2 * xi * xi - 0.5 * yi * xi
+            yn = yi
+            self.assertAlmostEqual(xn, xo)
+            self.assertAlmostEqual(yn, yo)
+
+        pinRoundTrip = pm.tranInverse(pout)
+        npt.assert_allclose(pin, pinRoundTrip, atol=1e-6)
+
+    def test_polyMapNoInverse(self):
+        """Test a unidirectional polymap with no numeric inverse
+        """
+        coeff_f = np.array([
+            [1.2, 1, 2, 0],
+            [-0.5, 1, 1, 1],
+            [1.0, 2, 0, 1],
+        ])
+        pm = astshim.PolyMap(coeff_f, 2, "IterInverse=0")
```

Comment:
You mean the `IterInverse=0` string? That's part of the AST API but it can also accept printf-style formatting. AST won't care if `astshim` builds the options internally (the Perl and `pyast` interfaces both support alternate ways of specifying the key/value options).